commands/general/contact.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    name: "contact",
+    description: "Where to contact your tutor.",
+    async execute() {
+        return "E-Mail: name@example.com\nZulip: John Doe\nMatrix: gt42qed@tum.de”";
```

Comment:
Are the duplicate ”" at the end of this line on purpose?

Reply:
@SimonPannek They are not. I guess thats what you get for thinking "There can't possibly be a mistake here, I don't have to test this"

Reply:
In my defense, my IDE syntax highlighting was drunk and seemed to have missed one of the quotation marks until I reloaded it.